docs/axes/cartesian/README.md

```diff
@@ -31,6 +31,7 @@ The following options are common to all cartesian axes but do not apply to other
 | `maxRotation` | `Number` | `90` | Maximum rotation for tick labels when rotating to condense labels. Note: Rotation doesn't occur until necessary. *Note: Only applicable to horizontal scales.*
 | `minRotation` | `Number` | `0` | Minimum rotation for tick labels. *Note: Only applicable to horizontal scales.*
 | `mirror` | `Boolean` | `false` | Flips tick labels around axis, displaying the labels inside the chart instead of outside. *Note: Only applicable to vertical scales.*
+| `offset` | `Boolean` | `false` | If true, extra space is added to the both edges and the axis is scaled to fit into the chart area. This is set to `true` in the bar chart by default.
```

Comment:
Here's a couple screenshots to show the `scale.bounds` option

**`scale.bounds: 'data'`**

![bounds-data](https://user-images.githubusercontent.com/322311/28998830-78e898a0-79e9-11e7-9ad7-0e41c743cb7c.png)


**`scale.bounds: 'ticks'`**

![bounds-ticks](https://user-images.githubusercontent.com/322311/28998835-83ee4e84-79e9-11e7-8192-58c7a2917e00.png)